### PR TITLE
Proposing to clarify

### PR DESCRIPTION
**Pull request by get-itips**
@carolynblanding Carolyn, please kindly review if you agree with these edits, the idea is to clarify the meaning of "re-image" and include some guidance on cleaning SfB Server (without undoing schema changes needed for Method 1)

For more information: 
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7249

@scanum please do not merge without Carolyn's approval.

@dariomws would like to have your opinion too.

### CODE REVIEW

**Comment by carolynblanding** [comment]
Looping in @João ***@***.***> for his review here.

Regards,
Carolyn Blanding

From: Andres Mariano Gorzelany ***@***.***>
Sent: Monday, May 24, 2021 11:04 AM
To: MicrosoftDocs/OfficeDocs-SkypeForBusiness ***@***.***>
Cc: Carolyn Blanding ***@***.***>; Mention ***@***.***>
Subject: [MicrosoftDocs/OfficeDocs-SkypeForBusiness] Proposing to clarify (#7291)


@carolynblanding<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcarolynblanding&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250115707%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=cfa%2B1q9IQrIO6%2BLWZSY0her30D9nkbtbznHRrNnZwKs%3D&reserved=0> Carolyn, please kindly review if you agree with these edits, the idea is to clarify the meaning of "re-image" and include some guidance on cleaning SfB Server (without undoing schema changes needed for Method 1)

For more information:
closes #7249<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fissues%2F7249&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250115707%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=sYxte3qULx6zCVvkjx9y%2FWHBqfzABo%2FSP9QZNA49p9k%3D&reserved=0>

@scanum<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fscanum&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250125664%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=xIVpKAKekLN1AQCIz%2BFABZXDZk8RE3CKWeRseJWsTeE%3D&reserved=0> please do not merge without Carolyn's approval.

@dariomws<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdariomws&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250125664%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=3HtaDyulmt7CReyo4jAoWSNT5luoSgH96cRGFzPM3cg%3D&reserved=0> would like to have your opinion too.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7291<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250125664%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=csiUiOJHZVyJIKK7tlbjyChR6BHTqhFdqgqEe9360R0%3D&reserved=0>

Commit Summary

  *   Proposing to clarify

File Changes

  *   M Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291%2Ffiles%23diff-32c25622ed031f714da2799efcd9497a03f19ab942bd0a2f3e6c1a9dbbd12672&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250135626%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ZtHF2AAG6XL8PUvuqfGZPa2JMsJ4NqbMUezhWbSlOdU%3D&reserved=0> (7)

Patch Links:

  *   https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7291.patch<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291.patch&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250135626%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=qtaWnpbpzHl4QvfJkf8saFKPpSjk4dSMxtkioyfd4zU%3D&reserved=0>
  *   https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7291.diff<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291.diff&data=04%7C01%7Ccbland%40microsoft.com%7C002ef6819bad4af53e7c08d91ec51f07%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574654250145581%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=vE1ItBhIGPM6A%2Fj2BI1zq0pstDX5gIbP8EpGrWh%2B5UM%3D&reserved=0>

-

**Comment by dariomws** [comment]
@get-itips agree with this PR. I understand the steps are the same except for 5 (Disable-CsAdDomain) and 6 (Disable-CsAdForest).

**Comment by carolynblanding** [comment]
Hi @Carolyn ***@***.***>

               Just a few comments on this statement

(…)Instead, you can follow this procedure without running steps 5 & 6 from Remove your on-premises Skype for Business deployment section (without undoing Forest-level and Domain-level changes)(…)

Assuming customer used method 1 get-csuser output is not empty (online homed users are returned). Have you confirmed this does not break in step 3 (Publish-CsTopology -FinalizeUninstall) due these objects being identified as SIP enabled users?
I am trying to avoid testing this as that will break my lab, so only as last resort option 😊

Regards,
João

From: Carolyn Blanding ***@***.***>
Sent: 24 de maio de 2021 18:29
To: MicrosoftDocs/OfficeDocs-SkypeForBusiness ***@***.***>; MicrosoftDocs/OfficeDocs-SkypeForBusiness ***@***.***>; João Loureiro ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [MicrosoftDocs/OfficeDocs-SkypeForBusiness] Proposing to clarify (#7291)

Looping in @João ***@***.***> for his review here.

Regards,
Carolyn Blanding

From: Andres Mariano Gorzelany ***@***.******@***.***>>
Sent: Monday, May 24, 2021 11:04 AM
To: MicrosoftDocs/OfficeDocs-SkypeForBusiness ***@***.******@***.***>>
Cc: Carolyn Blanding ***@***.******@***.***>>; Mention ***@***.******@***.***>>
Subject: [MicrosoftDocs/OfficeDocs-SkypeForBusiness] Proposing to clarify (#7291)


@carolynblanding<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcarolynblanding&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442661942%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5ZY6Z2eAbIjEju7%2FKa13IPLRdDRh07wZRDfvQDTgKAY%3D&reserved=0> Carolyn, please kindly review if you agree with these edits, the idea is to clarify the meaning of "re-image" and include some guidance on cleaning SfB Server (without undoing schema changes needed for Method 1)

For more information:
closes #7249<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fissues%2F7249&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442671899%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=NpZXgymnSbTryOqEE5p5XMKWRRpHjCitESneqbHTUdc%3D&reserved=0>

@scanum<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fscanum&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442681857%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=uYDqD5YMiBmWv%2B5jwyAtLYVleUMPu03wk0jzisBivro%3D&reserved=0> please do not merge without Carolyn's approval.

@dariomws<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdariomws&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442681857%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4z2szTYysgkbDKJ70nw88CDqr4r7Friuex1ZUCBz76Y%3D&reserved=0> would like to have your opinion too.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7291<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442691813%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=N2GzwpiXgWa1%2FL%2BcqLCA%2FLNPeiGqcVwalqWqxq28e08%3D&reserved=0>

Commit Summary

  *   Proposing to clarify

File Changes

  *   M Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291%2Ffiles%23diff-32c25622ed031f714da2799efcd9497a03f19ab942bd0a2f3e6c1a9dbbd12672&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442691813%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=xAgREtXSgMfW0YQbThQ9dk9MJvdiFhlQmS80rb62fv8%3D&reserved=0> (7)

Patch Links:

  *   https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7291.patch<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291.patch&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442851116%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=e7zXEf4EFwf4yUOMymk%2BgMtAAIcm%2FJNp6WGdCnnrnKA%3D&reserved=0>
  *   https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/7291.diff<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2FOfficeDocs-SkypeForBusiness%2Fpull%2F7291.diff&data=04%7C01%7Cjoaol%40microsoft.com%7C5e3911198b7d4ee4d50508d91ed96c0f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637574741442851116%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=lsf%2FKv%2FwO1GaFENy%2Bpmckgi5ek8TzJ9hDOj1CCJfScQ%3D&reserved=0>

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md [hunk 1]
```diff
@@ -113,6 +113,9 @@ After completing all the preliminary steps, you can remove the Skype for Busines
    Remove-CsConfigurationStoreLocation
    ``` 
 
+   > [!NOTE]
+   > If you are using Method 1 for managing user attributes, as described [here](cloud-consolidation-disabling-hybrid#method-1---manage-sip-addresses-and-phone-numbers-for-users-in-active-directory), do not continue with following steps.
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md:117]:
```suggestion
   > If you are using Method 1 for managing user attributes, as described [here](cloud-consolidation-disabling-hybrid#method-1---manage-sip-addresses-and-phone-numbers-for-users-in-active-directory), do not continue with the following steps.
```

File: Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md [hunk 2]
```diff
@@ -34,8 +34,8 @@ This article describes how to remove your on-premises Skype for Business deploym
 
 
 > [!IMPORTANT] 
-> The steps in this article apply only if you are using Method 2 for managing user attributes, as described [here](cloud-consolidation-disabling-hybrid.md#method-2---clear-skype-for-business-attributes-for-all-on-premises-users-in-active-directory). 
-If you are using Method 1, do not use the steps described in this article to remove your Skype for Business servers. Instead, you can re-image the servers.
+> The exact steps in this article apply only if you are using Method 2 for managing user attributes, as described [here](cloud-consolidation-disabling-hybrid.md#method-2---clear-skype-for-business-attributes-for-all-on-premises-users-in-active-directory). 
+If you are using Method 1, do not complete the whole procedure described in this article to remove your Skype for Business servers. Instead, you can follow this procedure without running steps 5 & 6 from Remove your on-premises Skype for Business deployment section (without undoing Forest-level and Domain-level changes).
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md:38]:
```suggestion
If you are using Method 1, do not complete the whole procedure described in this article to remove your Skype for Business servers. Instead, you can follow this procedure without running steps 5 and 6 from the _Remove your on-premises Skype for Business deployment_ section (without undoing Forest-level and Domain-level changes). 
```

File: Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md (hunk 1 above)
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/decommission-remove-on-prem.md:117]:
```suggestion
   > If you are using Method 1 for managing user attributes, as described [here](cloud-consolidation-disabling-hybrid#method-1---manage-sip-addresses-and-phone-numbers-for-users-in-active-directory), do not continue with the following steps.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestions only.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@carolynblanding PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
Hello @carolynblanding converting to draft to avoid an accidental merge, if Joao has doubts about step 3 (Publish-CsTopology -FinalizeUninstall), I could move the note just before that step, so at least we could remove the conference directories.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy is fine.

**Comment by get-itips** [comment]
> Hello @carolynblanding converting to draft to avoid an accidental merge, if Joao has doubts about step 3 (Publish-CsTopology -FinalizeUninstall), I could move the note just before that step, so at least we could remove the conference directories.

@carolynblanding Carolyn, how are you? returning to check if you could see my last message here, thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
This PR has conflicts that I am unable to resolve.  Also, the final reviewer should be @carolynblanding--not @CarolynRowe

**Pull request closed without merging**